pkg/integrations/integrations.go

```diff
@@ -23,11 +23,11 @@ type IntegrationConfig interface {
 }
 
 func GetIntegrationSnippet(intCfg IntegrationConfig, username string) (*string, error) {
-	if intCfg == nil {
+	userDatatValues := intCfg.GetUserDataValues()
```

Comment:
will it break if `intCfg` is nil, i am not sure if we can call this method `intCfg.GetUserDataValues()` when `intCfg` is nil

Reply:
This do not break, cause it is an interface, and interface always != nil, even when the content inside is nil (mean the struct which should implement the interface) this was the main issue around this. Probably we can refactor a bit better allowing only 1 integration per execution. 

But at least this does not break...as currently main is breaking

Reply:
nvm, i think its fine, since we always create a var in the package, it never nil

Reply:
> Probably we can refactor a bit better allowing only 1 integration per execution.

i was thinking to enforce this at the flag/cmd level.. preventing to pass both github and cirrus integration flags together.. add a validation to check only one integration flags are passed

Reply:
Yeah well we can do couple of things....

at logic level (pkg integrations... just return 1 integration snippet) 
at cmd level we can only allow flags for one or we can crate a common set flags (i.e. --integration-target (gh / cirrus), --integration-token, --integration-repo (if required in case you are using gh as target)
with this approach you already ensures only one integration can be set